pkg/shared/ruleset/disak8sstig/rules/242418.go

```diff
@@ -67,15 +70,27 @@ func (r *Rule242418) Run(ctx context.Context) (rule.RuleResult, error) {
 		return rule.Result(r, rule.WarningCheckResult(fmt.Sprintf("Option %s has been set more than once in container command.", option), target)), nil
 	}
 
-	ciphers := strings.Split(optSlice[0], ",")
-	requiredCiphers := []string{
-		"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
-		"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
-		"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
-		"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+	var (
+		ciphers         = strings.Split(optSlice[0], ",")
+		requiredCiphers = []string{
+			"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+			"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+			"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+			"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+		}
+		insecureCiphers = []string{
```

Comment:
Let's use the "unallowed" word since some of the ciphers are not deemed as insecure. 
```suggestion
		unallowedCiphers = []string{
```